tests/integration/purpose_test.py

```diff
@@ -24,22 +24,21 @@
 @pytest.fixture(scope="module")
 def atlan_tag(
     client: AtlanClient,
-) -> Generator[ClassificationDef, None, None]:
-    classification_def = ClassificationDef.create(
+) -> Generator[AtlanTagDef, None, None]:
+    classification_def = AtlanTagDef.create(
```

Comment:
I'd:
1. Change the variable name to `atlan_tag_def` for consistency
2. Change the name of the enum from `AtlanClassificationColor` to `AtlanTagColor`

Reply:
done